app/models/card.rb

```diff
@@ -1,6 +1,77 @@
 require "super_memo"
 
 class Card < ActiveRecord::Base
+  include Swagger::Blocks
+  swagger_schema :Card do
+    key :id, :Card
+    key :required, [
+                     :id,
+                     :original_text,
+                     :translated_text,
+                     :review_date,
+                     :block_id
+                   ]
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.